packages/contracts/test/ubiquistick/LP.t.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import "forge-std/Test.sol";
+
+import "../../src/ubiquistick/LP.sol";
```

Comment:
I'm actually not sure the purpose of this file and deleted it on my refactoring branch.

Reply:
removed `LP.t.sol`